components/DocumentModels.vue

```diff
@@ -0,0 +1,150 @@
+<script setup lang="ts">
+import type { IAllDocumentModels, TrackModel } from "@bcc-code/bmm-sdk-fetch";
+import type { IDiscoverableGroup } from "~/composables/discover";
+import { breakpointsTailwind, useBreakpoints } from "@vueuse/core";
+
+const { t } = useI18n();
+
+const props = withDefaults(
+  defineProps<{
+    type?: "page" | "playlist" | "private-playlist";
+    models: IAllDocumentModels[];
+  }>(),
+  {
+    type: "page",
+  },
+);
+
+const convertModels = (models: IAllDocumentModels[]) => {
+  let currentSection: IDiscoverableGroup["items"] = [];
+  const result: IDiscoverableGroup[] = [];
+  models.forEach((el, i) => {
+    if (el.type === "section_header") {
+      currentSection = [];
+      result.push({
+        header: el,
+        items: currentSection,
+      });
+    } else if (i === 0) {
+      currentSection = [el];
+      result.push({
+        header: null,
+        items: currentSection,
+      });
+    } else {
+      currentSection.push(el);
+    }
+  });
+  return result;
+};
+
+const { setQueue } = useNuxtApp().$mediaPlayer;
+
+const playItem = (item: TrackModel, group: IDiscoverableGroup) => {
+  const items = group.items.filter((c): c is TrackModel => c.type === "track");
+  setQueue(
+    items,
+    items.findIndex((track) => track.id === item.id),
+  );
+};
+
+// this is only needed while it's hosted on bmm-web.brunstad.org (Actually, it might be needed for )
```

Comment:
If you mean Electron, then this might be counter-productive... See https://github.com/bcc-code/bmm-web/blob/main/electron/main.ts#L38-L44

Reply:
changed the behavior